RELEASE-NOTES.rst

```diff
@@ -0,0 +1,39 @@
+===============================
```

Comment:
We may need to add this new file to `MANIFEST.in`, either manually or via `*.rst`.